### PR TITLE
Test: Make POSTGIS_VERSION detection automatic

### PR DESCRIPTION
**Pull request by Algunenano**
Same idea as https://github.com/CartoDB/Windshaft/pull/627

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ OPT_CREATE_PGSQL=yes # create the PostgreSQL test environment
 OPT_DROP_REDIS=yes   # drop the redis test environment
 OPT_DROP_PGSQL=yes   # drop the PostgreSQL test environment
 OPT_COVERAGE=no      # run tests with coverage
-OPT_DOWNLOAD_SQL=yes # download a fresh copy of sql files
+OPT_DOWNLOAD_SQL=no  # download a fresh copy of sql files
```
Comment (dgaubert) [inline run_tests.sh]: Travis will need this set to `yes` to get sql functions.

Comment (enekid) [inline run_tests.sh]: Travis is failing. Could it be because of this?

Comment (oleurud) [reply run_tests.sh]: :+1:

Comment (Algunenano) [reply run_tests.sh]: Yeah, I shouldn't have push that change

Comment (Algunenano) [reply run_tests.sh]: Yeah, I shouldn't have pushed that change. I'm removing it now

**Comment by Algunenano** [comment]
> Apart from detecting PostGIS Version automatically, what do you want to achieve with this PR?

For this PR just that, basically so if you are running the tests locally and you have a compatible Postgis release all ST_AsMVT tests are run too.

**Pull request merged**